integration/spark/src/main/spark2.3/org/apache/carbondata/spark/adapter/CarbonToSparkAdapter.scala

```diff
@@ -17,12 +17,12 @@
 
 package org.apache.carbondata.spark.adapter
 
-import scala.collection.mutable.ArrayBuffer
+import scala.collection.mutable.{ArrayBuffer, WrappedArray}
 
 import org.apache.spark.sql.execution.datasources.{FilePartition, PartitionedFile}
 
 object CarbonToSparkAdapter {
   def createFilePartition(index: Int, files: ArrayBuffer[PartitionedFile]) = {
-    FilePartition(index, files.toArray.toSeq)
```

Comment:
better to modify CsvRDDHelper directly and remove adapter/CarbonToSparkAdapter.scala （both 2.3 and 2.4）